scripts/linters/pylint_extensions.py

```diff
@@ -1751,40 +1751,34 @@ class NewlineBelowClassDocstring(checkers.BaseChecker):
     }
 
     def visit_classdef(self, node):
-        """Process a module to ensure that there is a single newline below
-        class docstring.
+        """Visit each class definition in a module.
 
         Args:
-            node: astroid.scoped_nodes.Function. Node to access module content.
+            node: astroid.scoped_nodes.ClassDef. Node for a class definition
```

Comment:
Remove `.` after ClassDef*

Reply:
We need this here. This is a check I need to implement in the second milestone

Reply:
Oh yeah! Thanks for the info!